deploy/aws/tidb-cluster/pre_userdata

```diff
@@ -0,0 +1,30 @@
+# According to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html
+# NVMe device names are in the format /dev/nvme[0-26]n1
+# and when the device is parted, it has /dev/nvme[0-26]n1p[1-xxx]
+# if the disk is parted, we will ignore the entire device
+# and if the disk is already formatted, then it will have a blkid, it should also be ignored
+for i in `seq 0 26`; do
+    if [ -e "/dev/nvme${i}" ]; then # find the device
+	if [ -e "/dev/nvme${i}n1" ]; then
+	    if ls /dev/nvme${i}n1p* > /dev/null 2>&1; then
+		echo "disk /dev/nvme${i}n1 already parted, skipping"
+	    else
+		echo "disk /dev/nvme${i}n1 is not parted"
+		if ! blkid /dev/nvme${i}n1 > /dev/null; then
+		    echo "/dev/nvme${i}n1 not formatted"
+		    mkfs -t ext4 /dev/nvme${i}n1
+		    mkdir -p /mnt/local-ssd/ssd${i}
```

Comment:
We should used the uuid here.

Reply:
The NVMe ssd disks have fixed device name as described here https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html. But yes it's more robust to use uuid.

Reply:
Good point. It is possible that the local volume provisioner code could (in the future, even if it doesn't now) assume different things depending on whether it sees it mounted with a UUID.